src/deployment/zilla-plus-in-production/secure-private-access/README.md

```diff
@@ -40,10 +41,18 @@ Enables Kafka clients to securely access multiple Amazon MSK Serverless clusters
 
 ![One to Many Private Access Overview](/one_to_many.png)
 
+### Secure Private Access via AWS Client VPN
+
+Zilla Plus supports secure private access to Amazon MSK Serverless over a custom domain, using AWS Client VPN.
+
+This allows users to securely connect to MSK Serverless clusters from outside the VPC without exposing the clusters to the public internet.
+
+![Secure Private Access via AWS Client VPN](/private_access_client_vpn.png)
+
 You will need to choose a wildcard DNS pattern to use for intranet access to the brokers in your Kafka cluster. These wildcard DNS names must resolve to the IP address of the VPC Endpoint in the client VPC, which then routes traffic via the VPC Endpoint Service to the ZillaPlus Network Load Balancer (NLB).
 
 Additionally, the Zilla Plus proxy must also be configured with a TLS server certificate representing the same wildcard DNS pattern.
 
```

Comment:
```suggestion

After deployment, setup an AWS Client VPN Endpoint attached to the client VPC so that on-premises clients can connect via AWS Client VPN and access the MSK Serverless clusters.
```

Suggest something like the above to cover the extra steps of VPN setup to complete the scenario after deployment.